### DISCUSSION TITLE
[Feature Request] Plugin version dropdown.

### DISCUSSION THREAD

**Discussion by TheFern2** [grafana/grafana#67294] [Plugins]
I am wondering if this has been suggested or talked about before.  We are running into an issue where a plugin is updated too often and sometimes installing the latest introduces breaking changes. Is there a possibility of providing a drop down to select a version of a plugin.

**Comment by bhks** [comment]
At present, Grafana supports installing or overriding a specific plugin version via HTTP API as mentioned in the https://github.com/grafana/grafana/pull/33836 however the full integration with UI has not been yet completed. This limits the ability of customers to manage the plugin from UI itself.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82974. Feel free to continue the discussion around this there. Thank you!
